src/OmniSharp.Abstractions/Models/v1/TypeLookup/DocumentationComment.cs

```diff
@@ -175,6 +189,32 @@ private static string TrimStartRetainingSingleLeadingSpace(string input)
                 return input;
             return $" {input.TrimStart()}";
         }
+
+        //Gets the parameter documentation from this object
+        public string GetParameterText(string name)
+        {
+            var requiredParam = Array.Find(ParamElements, parameter => parameter.Name == name);
```

Comment:
You could do an expression body here:
`GetParameterText() => Array.Find(...)?.Documentation ?? string.Empty.